src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
+dotnet build System.Device.Gpio.sln
+```
+This builds the main System.Device.Gpio assembly together with its test assembly. Before running the tests, you need to:
+- Connect an LED in series with a matching resistor to BCM Pin 18 (physical Pin 12) 
+- Connect BCM Pins 12 and 16 (physical Pins 32 and 36) with a cable
+
+## Raspberry Pi driver tests
+After that, you can run the tests with the RaspberryPiDriver (which is the default low-level driver for the Raspberry Pi) like:
+```
+dotnet test --filter RaspberryPiDriverTests System.Device.Gpio.sln 
```

Comment:
So these instructions are for  doing everything on the Pi? That might be a bit costly as building in the Pi is very slow. When we do local testing usually we just build in either Windows or Linux dev machine, and then we copy the build output into the Pi.

Reply:
Yes, these instructions are for building directly on the Pi. Since running the tests needs to be done on the Pi anyway (for obvious reasons), I showed the most straight-forward way of getting there. There seem to be no documentation on how to build and deploy from Visual Studio as well (and that involves more steps and should be documented at the root level). 

Or did I miss a way of remotely executing the tests from the Desktop?

Reply:
On any platform: `dotnet publish -r linux-arm`
On Pi or wherever you run you need to have runtime and you can: `dotnet xunit.console.runner.dll test.dll`
We currently don't have xunit runner dependency in the project but I'll add it when I'll be upgrading the setup

Reply:
I guess we can update/extend the documentation later, then?

Reply:
fair enough

Reply:
Actually seems you've already done that